vega_sim/scenario/common/agents.py

```diff
@@ -6,6 +6,7 @@
 from typing import List, Optional, Union
 from numpy.typing import ArrayLike
 from vega_sim.api.data import Order
+from vega_sim.api.helpers import num_to_padded_int
```

Comment:
Is this actually used anywhere? (hopefully isn't needed :) )

Reply:
Yep will remove, was used at one point but don't need anymore